expyfun/stimuli/_tracker.py

```diff
@@ -254,6 +262,31 @@ def respond(self, correct):
             self._x = np.append(self._x, self._x[-1] +
                                 self._current_step_size_up)
 
+        if bound:
+            if self._x_min is not None:
+                if self._x[-1] < self._x_min:
+                    self._limit_count += 1
+                    bad = True
+                    if self._repeat_limit == 'reversals':
+                        reversal = True
+                        self._n_reversals += 1
```

Comment:
The only reason I hadn't done that is because of the conditional for whether the tracker was stopped. Which on further inspection I'm not really sure why it's there? The next x value obviously won't be used, but it's added to ``self._x`` in lines 256-263 anyway so there's no reason _not_ to constrain it to the ``x_min`` or ``x_max``, right?